runners/core-java/src/main/java/org/apache/beam/runners/core/metrics/MonitoringInfoConstants.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.runners.core.metrics;
+
+import static org.apache.beam.model.pipeline.v1.MetricsApi.labelProps;
+import static org.apache.beam.model.pipeline.v1.MetricsApi.monitoringInfoSpec;
+
+import org.apache.beam.model.pipeline.v1.MetricsApi.MonitoringInfo;
+import org.apache.beam.model.pipeline.v1.MetricsApi.MonitoringInfo.MonitoringInfoLabels;
+import org.apache.beam.model.pipeline.v1.MetricsApi.MonitoringInfoSpecs;
+import org.apache.beam.model.pipeline.v1.MetricsApi.MonitoringInfoTypeUrns;
+import org.apache.beam.model.pipeline.v1.RunnerApi;
+
+/** This static class fetches MonitoringInfo related values from metrics.proto. */
+public final class MonitoringInfoConstants {
+
+  /** Supported MonitoringInfo Urns. */
```

Comment:
Can we make this multiple classes instead, which will shorten the use a bit
MonitoringInfoConstants.Urns.ELEMENT_COUNT
becomes
MonitoringInfoUrns.ELEMENT_COUNT

MonitoringInfoConstants.Labels.PTRANSFORM
becomes
MonitoringInfoLabels.PTRANSFORM

MonitoringInfoConstants.TypeUrns.SUM_INT64
becomes
MonitoringInfoTypeUrns.SUM_INT64

Reply:
I believe all of these have tight relation through MonitoringInfo.
Having them in single file/namespace is convenient to highlight the use case.

Also, I don't feel we win much of shorter usage. To make thinks even shorter, you can import sub-class directly and then it will be: Urls.ELEMENT_COUNT, etc.

Reply:
I have $0.02: group things according to functionality not what type of thing they are.

So, do not collocate these things because they are constants - collocate them because they are for working with `MonitoringInfo` objects. It is somewhat traditional at this point to use a plural like `MonitoringInfos` for utility methods and constants. It is not necessary, and IMO not useful, to distinguish constants from other static utility bits. I think these being collocated does make sense, but that is up to style.

Reply:
I think it makes sense to have all of these in one file. They're not so many that it's hard to track. I think either approach is acceptable though, so I wouldn't block the change either way.